ansible/files/bin/create-invite

```diff
@@ -6,7 +6,8 @@ if [ "$1" == "--qr" ]; then
 	shift;
 fi
 
-URL=$(prosodyctl mod_invites generate "$SNIKKET_DOMAIN" "$@")
+DOMAIN=${SNIKKET_TWEAK_XMPP_DOMAIN:-SNIKKET_DOMAIN}
```

Comment:
Won't this become literally `SNIKKET_DOMAIN` if the tweak variable is unset?

Reply:
Yes, that is the idea. 

On the base branch (snikket-im:split-xmpp-domain) when SNIKKET_TWEAK_XMPP_DOMAIN is set, Prosody uses that as the host and does not recognize SNIKKET_DOMAIN. 

```
Failed to execute command: Error: /usr/lib/prosody/modules/mod_invites.lua:245: Host ${SNIKKET_DOMAIN} does not exist
```

Reply:
I mean, the literal string `"SNIKKET_DOMAIN"`, not the variable.